tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json

```diff
@@ -5,8 +5,8 @@
     /* Projects */
     // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
     // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
-    // "tsBuildInfoFile": "./",                          /* Specify the folder for .tsbuildinfo incremental compilation files. */
-    // "disableSourceOfProjectReferenceRedirect": true,  /* Disable preferring source files instead of declaration files when referencing composite projects. */
+    // "tsBuildInfoFile": "./.tsbuildinfo",              /* Specify the path to .tsbuildinfo incremental compilation files. */
```

Comment:
i think it should be `file` instead of `files`

Reply:
originally it was files, it must be something else, I did pull to main and ran `gulp baseline-accept` and it still failed.

Reply:
You have to run the tests again before doing `gulp baseline-accept`

Reply:
I agree the message should say `file`